CONTRIBUTING.adoc

```diff
@@ -81,6 +81,8 @@ The wiki pages https://github.com/spring-projects/spring-framework/wiki/Code-Sty
 
 To format the code as well as check the style, run `./gradlew format check`.
 
+Lastly, run `./gradlew checkstyleMain checkstyleTest` before submitting a pull request, for additional style checks.
```

Comment:
Thank you for this polish. Will you please put in a separate commit so that the feature commit only includes the code needed for the feature?